scalatest/src/main/scala/org/scalatestplus/testng/TestNGSuiteLike.scala

```diff
@@ -13,19 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.scalatest.testng
+package org.scalatestplus.testng
 
 import org.scalatest._
 import org.scalatest.events._
 import exceptions._
 import org.scalactic.source
 import org.testng.TestListenerAdapter
 import org.testng.TestNG
-import Suite.formatterForSuiteAborted
-import Suite.formatterForSuiteCompleted
-import Suite.formatterForSuiteStarting
-import Suite.getIndentedTextForTest
-import Suite.wrapReporterIfNecessary
+import org.scalatest.ScalaTestInternals.Suite
+import org.scalatest.ScalaTestInternals.Suite.formatterForSuiteAborted
+import org.scalatest.ScalaTestInternals.Suite.getIndentedTextForTest
+import org.scalatest.ScalaTestInternals.Suite.wrapReporterIfNecessary
+import org.scalatest.ScalaTestInternals.Suite.yeOldeTestNames
+import org.scalatest.ScalaTestInternals.ScalaTestStatefulStatus
+import org.scalatest.ScalaTestInternals.Resources
 import events.MotionToSuppress
```

Comment:
We should copy these guys over to the org.scalatestplus.testng package and get rid of ScalaTestInternals, which is private[org].